src/SectionsNavigation.Uno.WinUI/Animations.cs

```diff
@@ -0,0 +1,176 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Media;
+using Microsoft.UI.Xaml.Media.Animation;
+
+namespace Chinook.SectionsNavigation
+{
+    /// <summary>
+    /// This class regroups helper methods to build animations.
+    /// </summary>
+    public static class Animations
```

Comment:
What are the differences between the UWP and WinUI versions? It might be better to share code instead of duplicating it.

Reply:
The main difference in most files are the using statements. Otherwise a few files are using DispatcherQueue instead of Dispatcher (with the extensions) on the frame. I will put those where I only need to change the using in a shared project with #if WINUI. Should I keep the ones with dispatcher changes separated?